parachains/runtimes/assets/statemine/src/xcm_config.rs

```diff
@@ -360,6 +361,8 @@ pub type Barrier = DenyThenTry<
 				AllowExplicitUnpaidExecutionFrom<ParentOrParentsPlurality>,
 				// Subscriptions for version tracking are OK.
 				AllowSubscriptionsFrom<ParentOrSiblings>,
+				// Allow unpaid execution when from a computed origin or plurarity on the relay chain.
+				AllowUnpaidExecutionFrom<ParentOrParentsPlurality>,
```

Comment:
@tonyalaribe 
what xcm transactions/scenarios do you have? Isnt it possible to extend them with `UnpaidExecution` instruction?

Reply:
My final XCM includes the `UnpaidExecution` instruction
```rust
Xcm(
      [
            SetAppendix(Xcm([ReportError(QueryResponseInfo { destination: MultiLocation { parents: 1, interior: Here }, query_id: 9, max_weight: <wasm:stripped> })])), 
            UnpaidExecution { weight_limit: Unlimited, check_origin: None }, DescendOrigin(X1(AccountId32 { network: None, id: [] })), 
           TransferAsset { 
                    assets: MultiAssets([MultiAsset { id: Concrete(MultiLocation { parents: 1, interior: Here }), fun: Fungible(5) }]), 
                    beneficiary: MultiLocation { parents: 0, interior: X1(AccountId32 { network: None, id: [] })}
             }
      ])

```

But that existing Barrier `AllowExplicitUnpaidExecutionFrom<ParentOrParentsPlurality>,` doesn't match it because the barrier expects the UnpaidExecution to be the first instruction which is not possible since SetAppendix always takes the first position in my scenario

Reply:
![image](https://user-images.githubusercontent.com/6564482/233982539-93c20c62-152d-4f19-b80f-a95339c6c956.png)


But I imagine a likely better solution could be for me to modify AllowExplicitUnpaidExecutionFrom to support the `UnpaidExecution` instructions being in other position (not position 1). I'll explore this and see if I can get it to work